source/classification1.md

```diff
@@ -388,13 +398,10 @@ Scatter plot of concavity versus perimeter with new observation represented as a
 ```{code-cell} ipython3
 :tags: [remove-cell]
 
-near_neighbor_df = pd.concat(
-    (
-        cancer.loc[np.argmin(my_distances), attrs],
-        perim_concav_with_new_point_df.loc[len(cancer), attrs],
-    ),
-    axis=1,
-).T
+near_neighbor_df = pd.concat([
+    cancer.loc[[np.argmin(my_distances)], attrs],
+    perim_concav_with_new_point_df.loc[[cancer.shape[0]], attrs],
+])
```

Comment:
Due to a small regression in altair 5.1 (to be fixed in 5.1.1) "object" columns are not longer automatically converted to quantitative types. We had some odd concat code that created objects where ints should be which I fixed, plus added some explicit types

Reply:
@joelostblom is this diff trying to match the temporarily broken altair 5.1, or the new fixed 5.1.1? Assuming 5.1.1 is coming soon, I'd prefer to just wait to merge this until 5.1.1 and follow that syntax.

Reply:
perhaps I am not understanding your comment well...is this new code here related to the regression in altair?